pkg/api/v1beta1/types.go

```diff
@@ -241,6 +241,16 @@ const (
 	PullIfNotPresent PullPolicy = "PullIfNotPresent"
 )
 
+type CapabilityType string
+
+// Capabilities represent POSIX capabilities that can be added or removed to a running container.
+type Capabilities struct {
+	// Added capabilities
+	Add []CapabilityType `json:"add,omitempty"`
```

Comment:
We have a presubmit check that verifies that API fields have descriptions. I can't merge this until they are added.